diff -uprN dropbear-2016.74/debug.h dropbear-2016.74-seaos/debug.h
--- dropbear-2016.74/debug.h	2016-09-25 22:54:25.232577102 -0700
+++ dropbear-2016.74-seaos/debug.h	2016-09-25 22:54:42.929039533 -0700
@@ -39,7 +39,7 @@
  * Caution: Don't use this in an unfriendly environment (ie unfirewalled),
  * since the printing may not sanitise strings etc. This will add a reasonable
  * amount to your executable size. */
-/*#define DEBUG_TRACE*/
+#define DEBUG_TRACE 1
 
 /* All functions writing to the cleartext payload buffer call
  * CHECKCLEARTOWRITE() before writing. This is only really useful if you're
diff -uprN dropbear-2016.74/includes.h dropbear-2016.74-seaos/includes.h
--- dropbear-2016.74/includes.h	2016-07-21 08:17:09.000000000 -0700
+++ dropbear-2016.74-seaos/includes.h	2016-09-25 15:21:24.263916592 -0700
@@ -156,11 +156,6 @@ typedef unsigned int u_int32_t;
 typedef u_int32_t uint32_t;
 #endif /* HAVE_UINT32_T */
 
-#ifdef SO_PRIORITY
-#include <linux/types.h>
-#include <linux/pkt_sched.h>
-#endif
-
 #include "fake-rfc2553.h"
 
 #ifndef LOG_AUTHPRIV
diff -uprN dropbear-2016.74/netio.c dropbear-2016.74-seaos/netio.c
--- dropbear-2016.74/netio.c	2016-07-21 08:17:09.000000000 -0700
+++ dropbear-2016.74-seaos/netio.c	2016-09-25 15:21:39.627091816 -0700
@@ -306,9 +306,6 @@ void set_sock_priority(int sock, enum dr
 #ifdef IPTOS_LOWDELAY
 	int iptos_val = 0;
 #endif
-#ifdef SO_PRIORITY
-	int so_prio_val = 0;
-#endif
 
 
 	/* Don't log ENOTSOCK errors so that this can harmlessly be called
@@ -333,19 +330,6 @@ void set_sock_priority(int sock, enum dr
 	}
 #endif
 
-#ifdef SO_PRIORITY
-	if (prio == DROPBEAR_PRIO_LOWDELAY) {
-		so_prio_val = TC_PRIO_INTERACTIVE;
-	} else if (prio == DROPBEAR_PRIO_BULK) {
-		so_prio_val = TC_PRIO_BULK;
-	}
-	/* linux specific, sets QoS class. see tc-prio(8) */
-	rc = setsockopt(sock, SOL_SOCKET, SO_PRIORITY, (void*) &so_prio_val, sizeof(so_prio_val));
-	if (rc < 0 && errno != ENOTSOCK)
-		dropbear_log(LOG_WARNING, "Couldn't set SO_PRIORITY (%s)",
-				strerror(errno));
-#endif
-
 }
 
 /* Listen on address:port. 
diff -uprN dropbear-2016.74/options.h dropbear-2016.74-seaos/options.h
--- dropbear-2016.74/options.h	2016-07-21 08:17:09.000000000 -0700
+++ dropbear-2016.74-seaos/options.h	2016-09-25 22:46:10.285080057 -0700
@@ -352,5 +352,5 @@ be overridden at runtime with -I. 0 disa
 /* Some other defines (that mostly should be left alone) are defined
  * in sysoptions.h */
 #include "sysoptions.h"
-
+#undef  DROPBEAR_SERVER_TCP_FAST_OPEN
 #endif /* DROPBEAR_OPTIONS_H_ */
